roles/edpm_pre_adoption_validation/molecule/adoption-negative/molecule.yml

```diff
@@ -0,0 +1,32 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: podman
+platforms:
+- command: /sbin/init
```

Comment:
nit: the convention for molecule is to use - name as the first key in each object in the list. rather then sorting the keys alphabetically.

it does not actually matter it just make multi instance test simpler to read

```
platforms:
  - name: edpm-0
    image: ...
    ...
```

Reply:
moved the name to the top here and in the other molecule.yml files